tests/e2e/expected_results/micronaut-core/micronaut-core.json

```diff
@@ -1523,8 +833,20 @@
+                    ],
+                    "result_type": "PASSED"
+                },
+                {
+                    "check_id": "mcn_provenance_witness_level_one_1",
+                    "check_description": "Check whether the target has a level-1 witness provenance.",
+                    "slsa_requirements": [
+                        "Provenance - Available - SLSA Level 1",
+                        "Provenance content - Identifies build instructions - SLSA Level 1",
+                        "Provenance content - Identifies artifacts - SLSA Level 1",
+                        "Provenance content - Identifies builder - SLSA Level 1"
                     ],
+                    "justification": [],
```

Comment:
Why is the `mcn_provenance_witness_level_one_1` passing for `micronaut-core`? :thinking:

Reply:
Previously, I did not return FAILED if there is no witness provenance discovered.
Fixed in d234c2736a370d13320b0b7338ba461224f2a31b.